packages/office-ui-fabric-react/src/components/Pivot/Pivot.Props.ts

```diff
@@ -50,6 +50,18 @@ export interface IPivotProps extends React.Props<Pivot> {
    */
   linkFormat?: PivotLinkFormat;
 
+  /**
+   * Specify whether to skip rendering the tabpanel with the content of the selected tab.
+   * Use this prop if you plan to separately render the tab content
+   * and don't want to leave an empty tabpanel in the page that may confuse Screen Readers.
+   */
+  headersOnly?: boolean;
```

Comment:
Do we want to close this up and put a backlog item in for splitting these up? Or is there still work being done on this PR?